testrunner/runBenchmark.js

```diff
@@ -130,7 +130,7 @@ export const runBenchmark = async (url) => {
 
     // set CPU to 6x slowdown
     const client = await page.context().newCDPSession(page);
-    await client.send('Emulation.setCPUThrottlingRate', { rate: 6 });
+    await client.send('Emulation.setCPUThrottlingRate', { rate: 20 });
```

Comment:
Please keep this at 6x for now to stay inline with Krauset and continuation of the previous results.